codespell_lib/_codespell.py

```diff
@@ -48,7 +50,7 @@
     ('usage', 'for recommended terms', '_usage', None, None),
     ('code', 'for words common to code and/or mathematics', '_code', None, None),  # noqa: E501
     ('names', 'for valid proper names that might be typos', '_names', None, None),  # noqa: E501
-    ('en-GB_to_en-US', 'for corrections from en-GB to en-US', '_en-GB_to_en-US', True, True),  # noqa: E501
+    ('en-GB_to_en-US', 'for corrections from en-GB to en-US', '_en-GB_to_en-US', ('en_GB-ise',), ('en_US',)),  # noqa: E501
```

Comment:
So err in aspell/correction in aspell can now be one of:
- None - Skip checking
- False - Not in supported_languages
- True - In supported_languages
- Array - In array languages

Personally I wonder if that's too much overloading, we've gone from a tristate to multi-state. I wonder if having an (optional) array of languages configured for each one and then the existing tristate might be clearer, i.e.:
```suggestion
    ('en-GB_to_en-US', 'for corrections from en-GB to en-US', '_en-GB_to_en-US', True, True, ('en_GB-ise',), ('en_US',)),  # noqa: E501
```

Or:
`('clear', 'for unambiguous errors', '', False, None, None, None),`
Which would be equivalent to:
`('clear', 'for unambiguous errors', '', False, None, supported_languages, None),`

Reply:
I can make it as an extra option too. That was my first idea.
But then I thought, that this could be to many options.

I would wait for the review of @larsoner and adapt it to the consensus.

Reply:
I don't think that matters too much on internal code (it's not like a user has to deal with this), indeed I'd think making the tests/config clear and removing all that extra logic is probably a good thing to make the tests more supportable.

Reply:
I like the idea for the optional array. 
One can implement the feature:
`
('not_british', 'For words, that are not used in british', '', False, None, ((en_GB,),  None),
`
This could be beneficial if multiple languages are supported.

Reply:
FWIW I'd probably go for two options, so as follows rather than your suggestion:
`('not_british', 'For words, that are not used in British English', '', False, None, (en_GB,), None),`

Unless you just added an extra comma by mistake. So a slightly more real world example would convert those not-British words to their British equivalents:
`('not_british', 'For words, that are not used in British English', '', False, True, (en_GB,), (en_GB,)),`

Sorry, I should have been clearer, only optional in the sense you could use None or a preset, but two arrays, one for the error and one for the corrections.

Reply:
Yes, your example is more realistic. I just wanted to give an example that came in 5 seconds into my mind.

And I had a syntax error in my quick example :smile:

Reply:
> And I had a syntax error in my quick example smile

I thought you did, but I wasn't sure, which was part of the reason for my alternative example.